frontend/src/pages/team/Brokers/Hierarchy/components/suggestions/TopicSuggestion.vue

```diff
@@ -0,0 +1,78 @@
+<template>
+    <li class="suggestion p-4 border border-gray-200 bg-white rounded-md flex gap-10 items-center justify-between">
+        <div class="content flex flex-col gap-1">
+            <div class="title-wrapper">
+                <span class="title">Message Format: </span>
+                <span class="format">{{ capitalize(format) }}</span>
+            </div>
+            <div class="description-wrapper">
+                <p v-if="description" class="description opacity-50 text-xs leading-none">{{ description }}</p>
+                <p v-else class="description opacity-50 text-xs leading-none">
+                    FlowFuse has detected that the messages sent to this topic are {{ format.toUpperCase() }}. Would you like to enforce this on your Schema?
```

Comment:
```suggestion
                    FlowFuse has detected that the messages sent to this topic are {{ format.toUpperCase() }}. Would you like to save this to your Schema?
```

We aren't "enforcing", it's guidance/documentation at this time